src/Symfony/Component/VarDumper/Resources/functions/dump.php

```diff
@@ -66,3 +66,70 @@ function dd(mixed ...$vars): never
         exit(1);
     }
 }
+
+if (!function_exists('dumph')) {
+    /**
+     * @author Nicolas Grekas <p@tchwork.com>
+     * @author Alexandre Daubois <alex.daubois@gmail.com>
+     */
+    function dumph(mixed ...$vars): mixed
+    {
+        // CORS for SPA apps like.
+        header("Access-Control-Allow-Origin: " . $_SERVER['HTTP_ORIGIN'] ?? '*');
+        header('Access-Control-Allow-Methods: *');
+        header('Access-Control-Allow-Headers: *');
+        header('Access-Control-Allow-Credentials: true');
+
+        if (!$vars) {
+            VarDumper::dump(new ScalarStub('🐛'));
+
+            return null;
+        }
+
+        if (array_key_exists(0, $vars) && 1 === count($vars)) {
+            VarDumper::dump($vars[0]);
+            $k = 0;
+        } else {
+            foreach ($vars as $k => $v) {
+                VarDumper::dump($v, is_int($k) ? 1 + $k : $k);
+            }
+        }
+
+        if (1 < count($vars)) {
+            return $vars;
+        }
+
+        return $vars[$k];
+    }
+}
+
+if (!function_exists('ddh')) {
+    function ddh(mixed ...$vars): never
+    {
+        // CORS for SPA apps like.
+        header("Access-Control-Allow-Origin: " . $_SERVER['HTTP_ORIGIN'] ?? '*');
+        header('Access-Control-Allow-Methods: *');
+        header('Access-Control-Allow-Headers: *');
+        header('Access-Control-Allow-Credentials: true');
```

Comment:
maybe? i am not sure, i never have problems with dd, expect SPA/javascript apps. 
i don't think adding directly to dd will negative impact any scenario, since is a debug function.
on my local machine i always add this to dd and i don't have any problems.